FrontEnd/FrontEnd.Web/Components/CustomIcons/CustomIcons.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.FluentUI.AspNetCore.Components;
+
+namespace FrontEnd.Web.Components.CustomIcons;
+
+public static class CustomIcons
```

Comment:
Probably would make more sense to just store this in [wwwroot](https://github.com/BenasB/vu-psk/tree/main/FrontEnd/FrontEnd.Web/wwwroot) as files

Reply:
@6ar8nas also thought maybe it's worth to just have a [`Menu`](https://www.fluentui-blazor.net/Menu) component with the title "Share" (and a share icon on the right). The menu could then just display the options in text: "Facebook", "Reddit", etc.

![image](https://github.com/BenasB/vu-psk/assets/29711974/19bc85f1-6513-4efb-a605-491887573157)

Reply:
Yeah, good point. Will address this shortly.

Reply:
FluentMenuItem only supports OnClick event handler. In that case, we would need to navigate with the current tab or inject JSRuntime to have everything working (that in turn would cause problems with popup blockers). Decided to instead workaround it by using FluentAnchor, which supports hrefs and blank target,